config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'articles/index'
```

Comment:
Please remove auto generated routes. Also,  follow the standard using `resources` if you need to generate routes for a resource/model.